system-test/fixtures/google-gax-packaging-test-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "google-gax-packaging-test-app",
+  "version": "0.0.1",
+  "description": "Test application for google-gax",
+  "main": "src/index.js",
+  "files": [
+    "src",
+    "protos"
+  ],
+  "license": "Apache-2.0",
+  "keywords": [],
+  "scripts": {
+    "lint": "eslint src/ samples/ system-test/ test/",
+    "prettier": "prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
+    "start": "node src/index.js",
+    "test": "mocha test/*.js"
+  },
+  "devDependencies": {
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^4.1.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-prettier": "^3.0.1",
+    "gts": "^1.0.0-0",
```

Comment:
just `1.0.0` now :)

Reply:
It's not TypeScript (yet) :(  It's a leftover. No need to have any kind of `gts` here.